test/splitgraph/commands/test_es_aggregation_pushdown.py

```diff
@@ -169,7 +282,61 @@ def test_elasticsearch_grouping_and_aggregations_bare(snapshot, local_engine_emp
 
 
 @pytest.mark.mounting
-def test_elasticsearch_agg_subquery_pushdown(local_engine_empty):
+def test_grouping_and_aggregations_filtering(snapshot, local_engine_empty):
+    _mount_elasticsearch()
+
+    # Aggregations functions and grouping bare combination
+    query = """
+    SELECT state, age, min(balance), COUNT(*)
+    FROM es.account
+    WHERE gender = 'M' AND age = ANY(ARRAY[25, 35])
+    GROUP BY state, age
+    """
+
+    # Ensure query is going to be pushed down
+    result = get_engine().run_sql("EXPLAIN " + query)
+    assert _extract_queries_from_explain(result)[0] == {
+        "query": {
+            "bool": {
+                "must": [
```

Comment:
Why the `should` nested inside of `must` -- is it so that ES treats the `should` clauses as "one of these must match"?

Reply:
Makes sense. I have this weird memory of Multicorn not supporting `OR` clauses, but `ANY` bypasses that.